e3db/src/main/java/com/tozny/e3db/Config.java

```diff
@@ -217,4 +217,32 @@ public String json() {
       throw new RuntimeException(e);
     }
   }
+
+  /**
+   * TODO
```

Comment:
We should either add docs to replace the `TODO`s here, or at least remove the `TODO`s.

Reply:
Documentation is on the TODO list.